pkgs/top-level/all-packages.nix

```diff
@@ -1402,6 +1402,8 @@ in
 
   clipster = callPackage ../tools/misc/clipster { };
 
+  contrast = callPackage ../applications/accessibility/contrast { };
```

Comment:
I think this name could be better TBH, but that's their choice.

Reply:
Yeah, I was surprised this name was not already taken in nixpkgs. And apparently `palette` is not either.

Reply:
seems at least elementary got it right https://github.com/danrabbit/harvey :smile: